geo-types/src/coordinate.rs

```diff
@@ -68,3 +69,12 @@ where
         (self.x, self.y)
     }
 }
+
+impl<T> fmt::Display for Coordinate<T>
```

Comment:
This was added mainly for hashing the coordinates (to a string, i.e. enabling the underlying `to_string` method) such that we can build up an index for the vertices, and retrieve them in O(1) time when queried. I'm not a super fan of this approach either, so I'm open to other workarounds. 

Alternatively, we can just loop over all the vertices to find an arbitrary vertex, though this may become a costly operation when dealing with a large graph.

Another approach would be hashing the [f64; 2] array, which is essentially what's behind the `Coordinate<T>` type. Yet we need to convert the inner coordinates to be `Float` or `f64` compatible.